infra/token-transfer-client/src/components/BonusModal.js

```diff
@@ -128,11 +148,15 @@ class BonusModal extends Component {
                 <div className="col">
                   <strong>Bonus tokens earned</strong>{' '}
                   <span style={{ fontSize: '14px' }}>
-                    ({bonusRate}% of lockup)
+                    ({lockupBonusRate}% of lockup)
                   </span>
                 </div>
                 <div className="col-4 text-right">
-                  <strong>{this.state.amount * (bonusRate / 100)}</strong>{' '}
+                  <strong>
+                    {BigNumber(this.state.amount * (lockupBonusRate / 100))
+                      .toFixed(0, 1)
```

Comment:
Per my other comment, I'd use toFixed(0, BigNumber.ROUND_UP).

Same comment for file LockupCard.js